### PR TITLE
Refactor code on the Appointments form

### PR DESCRIPTION
**Pull request by jwnasambu**
## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide).
- [ ] My work includes tests or is validated by existing tests.

## Summary
Fixed a ternary with two branches with the same text in each. I created a little mini-component for the location select and  wrapped it in a layer.

### CODE REVIEW

**Review by hadijahkyampeire — COMMENTED** [review]
Thanks for the refactoring @jwnasambu 👍 , but since you are refactoring this file, could you please also refactor the `DatePicker` select, it also has repeated code, check line 175.

**Comment by jwnasambu** [comment]
@hadijahkyampeire Thanks for the point out. I don't know how I missed it out.

**Comment by jwnasambu** [comment]
@hadijahkyampeire I have fixed the proposed changes.

File: packages/esm-patient-appointments-app/src/appointments/appointments-form.component.tsx [hunk 1]
```diff
@@ -173,49 +178,7 @@ const AppointmentsForm: React.FC<DefaultWorkspaceProps> = ({ patientUuid, closeW
...
-                  value={timeFormat}
-                  labelText={t('time', 'Time')}
-                  aria-label={t('time', 'Time')}
-                >
-                  <SelectItem value="AM" text="AM" />
-                  <SelectItem value="PM" text="PM" />
-                </TimePickerSelect>
-              </TimePicker>
-            )}
+            {isTablet ? <Layer> {timePicker} </Layer> : timePicker}
```
Comment (hadijahkyampeire) [inline packages/esm-patient-appointments-app/src/appointments/appointments-form.component.tsx:181]:
```suggestion
            {isTablet ? <Layer> {timePicker} </Layer> : timePicker}
            {isTablet ? <Layer>{timePicker}</Layer> : timePicker}
```
Just removing the unnecessary space.

**Review by hadijahkyampeire — APPROVED** [review]
LGTM, thanks @jwnasambu , minor: just remove the unnecessary space.

**Comment by jwnasambu** [comment]
@hadijahkyampeire its fixed! thanks

**Pull request merged**